### PR TITLE
made tablular_inline.html compatible with recent changes

### PR DESCRIPTION
**Pull request by preston-wagner**

### CODE REVIEW

File: viewsets/static/scripts/add_more_inlines.js [hunk 1]
```diff
@@ -1,13 +1,13 @@
 $(function(){
-    $('.form-inline > .add-more').click(function(event){
```
Comment (freakypie) [inline viewsets/static/scripts/add_more_inlines.js]: why bother adding .inline? why not just use .form-inline?

File: viewsets/static/scripts/add_more_inlines.js [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
+    $('.inline > .add-more').click(function(event){
         event.preventDefault();
         var $inline_form = $(this).parent();
         var $total_forms = $inline_form.find('[name*="TOTAL_FORMS"]');
         var form_idx = $total_forms.val();
         var $empty_form = $inline_form.find('.empty-form');
-        var $forms_list = $inline_form.find('.forms');
```
Comment (freakypie) [inline viewsets/static/scripts/add_more_inlines.js:8]: Why add the table:not selector? something to do with crispy forms?

File: viewsets/templates/forms/tabular_inline.html [hunk 3]
```diff
@@ -1,31 +1,12 @@
...
-	                        {% for field in form.hidden_fields %}
-	                            {{ field }}
-	                        {% endfor %}
-	                    {% endif %}
-	                    {{ field }}
-	                </td>{% endfor %}
-	            </tr>{% endfor %}
-	        </tbody>
-	    </table>
-    </div>
```
Comment (freakypie) [inline viewsets/templates/forms/tabular_inline.html:30]: I like the idea of moving to crispy forms

File: viewsets/templatetags/viewgroups_tags.py [hunk 4]
```diff
@@ -36,7 +36,9 @@ def form_helper(
         form_tag=False,
         label_size=4,
         label_offset=0,
-        field_size=8):
+        field_size=8,
+        template=None,
+):
```
Comment (freakypie) [inline viewsets/templatetags/viewgroups_tags.py:41]: template stuff is a good idea too

**Review by freakypie — COMMENTED** [review]
Please lemme know the answers to the questions I left, then i will reconsider

File: viewsets/static/scripts/add_more_inlines.js (hunk 1 above)
Comment (preston-wagner) [reply viewsets/static/scripts/add_more_inlines.js]: .form-inline is from bootstrap, and applies styling that you may not want added.

File: viewsets/static/scripts/add_more_inlines.js [hunk 5]
```diff
@@ -5,9 +5,9 @@ $(function(){
...
+        var $forms_list = $inline_form.find('.forms, table:not(.empty_form_table)>tbody');
```
Comment (freakypie) [inline viewsets/static/scripts/add_more_inlines.js:8]: OK so why is this selector needed?

**Pull request closed without merging**